Bluepill-cli/Bluepill-cli/Bluepill/Reporters/BPTreeParser.m

```diff
@@ -296,6 +296,9 @@ - (void)parseLine:(nullable NSString *)line {
                 testCaseLogEntry.lineNumber = [lineNumber integerValue];
                 testCaseLogEntry.unexpected = isError;
                 testCaseLogEntry.errorMessage = errorMessage;
+
+                // Report assertion failure
+                [self onTestCaseAssertionFailedWithName:testCaseLogEntry.testCaseName inClass:testCaseLogEntry.testCaseClass];
```

Comment:
just call `saveScreenshot` in delegate's onTestCaseFailedWithName function

Reply:
So we can assume, that onTestCaseFailedWithNamewill be called later yes?

Reply:
from that file, yes